### DISCUSSION TITLE
Is there a way to set autolinks at a org level instead of a repo level?

### DISCUSSION THREAD

**Discussion by winniengo** [community/community#30712] [Other Feature Feedback, Questions, & Ideas]
Bumping on a previously shared topic: https://github.com/orgs/community/discussions/12386

Does anyone know of a way to set autolinks (https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources) at an organization-level instead of only at a repository-level? It's very difficult to have to maintain when you have multiple teams, projects and and new repos popping up 😞

**Comment by mbonaci** [comment]
A dupe of https://github.com/orgs/community/discussions/12386
